packaging/prerelease/s3_index.sh

```diff
@@ -41,9 +41,3 @@ fi
 
 echo "Checking if we need to promote a release for testing ($platform)"
 "$release_bin" promote-test-releases --bucket-name="$bucket_name" --platform="$platform"
```

Comment:
Didn't feel like adding the code on the slackbot side, but if you want I can do that too.